pants

```diff
@@ -250,12 +253,25 @@ function bootstrap_pants {
 
   if [[ ! -d "${PANTS_BOOTSTRAP}/${target_folder_name}" ]]; then
     (
-      local venv_path
-      venv_path="$(bootstrap_venv)"
+      green "Bootstrapping Pants using ${python}"
       local staging_dir
       staging_dir=$(tempdir "${PANTS_BOOTSTRAP}")
+
+      if ${python} -c "import venv" > /dev/null  2>&1; then
```

Comment:
I think you want to quote `${python}` - people do strange things.

Reply:
Done.